aten/src/ATen/cpu/vec256/vec256_complex_float.h

```diff
@@ -241,10 +241,7 @@ template <> class Vec256<c10::complex<float>> {
     return Vec256(_mm256_permute_ps(ln.values, 0xB1)).conj();         //-i*ln()
   }
   Vec256<c10::complex<float>> acos() const {
-    // acos(x) = pi/2 - asin(x)
-    constexpr float pi_2f = c10::pi<float> / 2;
-    const __m256 pi_2 = _mm256_setr_ps(pi_2f, 0.0, pi_2f, 0.0, pi_2f, 0.0, pi_2f, 0.0);
-    return _mm256_sub_ps(pi_2, asin());
+    return map(std::acos);
```

Comment:
could you please benchmark the performance?

Reply:
It would be roughly 2x slower than current implementation, see document submitted for https://github.com/pytorch/pytorch/pull/28735: https://docs.google.com/spreadsheets/d/17pObc
rSTpV4BOOX9FYf1vIX3QUlEgQhLvL1IBEyJyzs